readme.md

```diff
@@ -142,6 +142,9 @@ a tailwind color. Especially useful if you're using a customized color pallette.
 
 ```js
 tw.color('blue-100');
+tw.color('bg-blue-100');
+tw.color('border-blue-100');
+tw.color('text-blue-100');
```

Comment:
I'm not sure I see the reason for showing incorrect usage in the readme. Can you explain why you think this is worthwhile?

Reply:
I would say that being explicit about what is possible when calling `tw.color` is better because else I can only know that I can pass `bg|border|text` if going through the source code of the lib 😅 

From my point of view, the "incorrect usages" are the most useful ones as I can reuse my classNames to extract colors